src/get.js

```diff
@@ -39,15 +39,16 @@ module.exports = configure(({ ky }) => {
     const extractor = Tar.extract()
 
     for await (const { header, body } of extractor(toIterable(res.body))) {
+      const file = {
+        path: header.name,
+        mtime: { secs: Math.floor(header.mtime.getTime() / 1000) },
```

Comment:
Can this support the `nsecs` mtime component too?  At the moment I think it's going to throw away anything that isn't a whole second.